stress/common/src/test/java/alluxio/stress/BaseParametersTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParametersDelegate;
+import org.junit.Test;
+
+import java.util.List;
+
+public class BaseParametersTest {
+  @ParametersDelegate
+  private BaseParameters mBaseParameter = new BaseParameters();
+
+  @Test
+  public void parseParametersToArgumentString() throws Exception {
```

Comment:
This test is good. Can you add more tests with different combinations, especially corner cases?